### DISCUSSION TITLE
What does the various features do? `track-caller`? `generate-parsing-rs`?

### DISCUSSION THREAD

**Discussion by CGMossa** [JelteF/derive_more#306] [Q&A]
There are features that I can only guess what they entail.

`track-caller`? `generate-parsing-rs`? 

I've tried to look, but cannot find a place where there is documentation of the present features

**Comment by tyranron** [comment]
@CGMossa as Cargo doesn't support public/private features, the common unspoken rule is that if the feature is not described explicitly - it's unlikely you need it, and it's likely an internal one.

`generate-parsing-rs` is an internal feature for generating `fmt` syntax parser code, when maintaining this crate.

`track-caller` is a feature which considers support for [`#[track_caller]` attribute](https://github.com/JelteF/derive_more/blob/v0.99.17/Cargo.toml#L69-L73) for Rust versions before 1.46.

Nevertheless, you shouldn't bother about them. Just consider using [latest `1.0.0-beta.*` releases](https://docs.rs/derive_more/1.0.0-beta.3). They doesn't contain such features anymore. And lot of stuff has changed since then, regarding ergonomics.
